omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/VisitController1_9Test.java

```diff
@@ -214,4 +214,14 @@ public void searchByPatient_shouldGetUnretiredVisitsForThePatient() throws Excep
 		Assert.assertEquals(3, Util.getResultsSize(result));
 	}
 	
+	@Test
+	public void searchByPatient_shouldGetAllVisitsForThePatientIncludingRetired() throws Exception {
```

Comment:
The test method name is a bit misleading because it conveys that this method should unconditionally include retired visits at all times.